Resources/Locale/en-US/escape-menu/ui/options-menu.ftl

```diff
@@ -112,13 +112,13 @@ ui-options-header-dev = Development
 ui-options-header-general = General
 
 ui-options-hotkey-keymap = Use US QWERTY Keys
-ui-options-hotkey-toggle-walk = Toggle Walk
+ui-options-hotkey-toggle-walk = Toggle Speed
```

Comment:
Not quite sure about this, same reason with it being an optional change. You could make two separate Loc strings, one for walk and one for sprint, that wouldn't be unreasonable.

Reply:
It's an engine bind, I cannot make it switch between two locales.